pillowtop/listener.py

```diff
@@ -77,12 +72,14 @@ def lock_manager(obj):
         return LockManager(obj, None)
 
 
-class BasicPillow(object):
+class BasicPillow(PillowBase):
+    """
+    BasicPillow is actually a CouchPillow. PillowBase defines the actual interface.
```

Comment:
Is this comment basically saying "BasicPillow" should actually be called "CouchPillow"? (And if so...)